src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+import '../../App.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {
+    person.age
+      && <p className="Person__age">{`I am ${person.age}`}</p>
+    }
+
+    {
+    person.isMarried ? <p className="Person__partner">{`${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`}</p>
```

Comment:
```suggestion
    isMarried ? (
      <p className="Person__partner">
        {`${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`}
      </p>
    ) 
```
fix indentation
and better destructure person to decrease this line

Reply:
pass the whole person as a prop <Person person={misha} /> (not individual fields);
it is from implementation task guide